### PR TITLE
Create a provider using `ethers.getDefaultProvider()`

### PR DESCRIPTION
**Pull request by Elemino**
Attempted to create the ethers.js provider but not sure if this should be declared globally and if so, where it should be declared. 

Once the provider is successfully instantiated, the next step should be to migrate all the read-only method calls for listing cards, adding wallet signers etc. 

I have a lot of gumption to keep working on this great project but might need a bit more guidance.

### CODE REVIEW

File: cryptoxmas-contracts/scripts/deploy.js [hunk 1]
```diff
@@ -6,11 +6,12 @@ import CryptoxmasEscrow from '../build/CryptoxmasEscrow';
 import NFT from '../build/NFT';
 import { deployContract, storeData, waitForTransactionReceipt } from './helpers';
 
+
 const deploy = async (network) => {
     
     console.log("Deploying contract for ", network);
     const networkConfig = config[network];
-    const provider = new providers.JsonRpcProvider(networkConfig.JSON_RPC_URL);
+    const provider = new ethers.providers.JsonRpcProvider(networkConfig.NFT_ADDRESS);
```
Comment (mikhaildobs) [inline cryptoxmas-contracts/scripts/deploy.js]: No need to make changes in cryptoxmas-contracts/scripts files

File: cryptoxmas-contracts/scripts/mintTokens.js [hunk 2]
```diff
@@ -15,11 +15,12 @@ import {
 } from './helpers';
 
 
+
 const mint = async (network, escrowAddress) => {
 
     const networkConfig = config[network];
     console.log("Minting tokens ", network);
-    const provider = new providers.JsonRpcProvider(networkConfig.JSON_RPC_URL);
+    const provider = new ethers.providers.JsonRpcProvider(networkConfig.NFT_ADDRESS);
```
Comment (mikhaildobs) [inline cryptoxmas-contracts/scripts/mintTokens.js]: No need to make changes in cryptoxmas-contracts/scripts files

**Comment by Dobrokhvalov** [comment]
> Attempted to create the ethers.js provider but not sure if this should be declared globally and if so, where it should be declared.

Let's make a separate ethersService like https://github.com/VolcaTech/cryptoxmas.xyz/blob/master/src/services/web3Service.js
with the similar `setup` method and `getProvider` instead of `getWeb3`

**Comment by Elemino** [comment]
Had the wrong conceptions of a few basic things but I'm getting there via trial and error. I expect to complete the PR in two days. Thanks.

**Pull request closed without merging**